### ISSUE TITLE
Opening public shared txt file causes internal server error

### ISSUE THREAD

**Issue by js94x** [nextcloud/richdocuments#131]
### Steps to reproduce
1. create txt file over web-gui
2. write content 
3. create a public share for the single file 
4. open the public link in another browser session

### Expected behaviour
The txt file should be editable/viewable via Collabora. 

### Actual behaviour
A error message appears instead and the file must be downloaded via the download button to view it.

Error message:
```
Exception: Could not find urlsrc in WOPI

    /www/apache/oc.xxxxxxx.org/htdocs/apps/richdocuments/lib/TokenManager.php - line 117: OCA\Richdocuments\WOPI\Parser->getUrlSrc('text/plain')
    /www/apache/oc.xxxxxxx.org/htdocs/apps/richdocuments/lib/Controller/DocumentController.php - line 242: OCA\Richdocuments\TokenManager->getToken(*** sensitive parameters replaced ***)
    [internal function] OCA\Richdocuments\Controller\DocumentController->publicPage('3mU3zXr7QR6quiC', 'test.txt', '{fileId}')
    /www/apache/oc.xxxxxxx.org/htdocs/lib/private/AppFramework/Http/Dispatcher.php - line 160: call_user_func_array(Array, Array)
    /www/apache/oc.xxxxxxx.org/htdocs/lib/private/AppFramework/Http/Dispatcher.php - line 90: OC\AppFramework\Http\Dispatcher->executeController(Object(OCA\Richdocuments\Controller\DocumentController), 'publicPage')
    /www/apache/oc.xxxxxxx.org/htdocs/lib/private/AppFramework/App.php - line 114: OC\AppFramework\Http\Dispatcher->dispatch(Object(OCA\Richdocuments\Controller\DocumentController), 'publicPage')
    /www/apache/oc.xxxxxxx.org/htdocs/lib/private/AppFramework/Routing/RouteActionHandler.php - line 47: OC\AppFramework\App main('OCA\\Richdocumen...', 'publicPage', Object(OC\AppFramework\DependencyInjection\DIContainer), Array)
    [internal function] OC\AppFramework\Routing\RouteActionHandler->__invoke(Array)
    /www/apache/oc.xxxxxxx.org/htdocs/lib/private/Route/Router.php - line 299: call_user_func(Object(OC\AppFramework\Routing\RouteActionHandler), Array)
    /www/apache/oc.xxxxxxx.org/htdocs/lib/base.php - line 1004: OC\Route\Router->match('/apps/richdocum...')
    /www/apache/oc.xxxxxxx.org/htdocs/index.php - line 48: OC handleRequest()
    {main}
```
### Server configuration
**Nextcloud version:** 12.0.3
**App version:** 1.12.35

**Comment by timtim8461** [comment]
Got the same problem. Did you fix the problem?

**Comment by timar** [comment]
Probably it was fixed by #170 
I don't see it with app version 2.0.2 on Nextcloud 13.
